web/geo/generate-aggregates.js

```diff
@@ -0,0 +1,51 @@
+const { union } = require('@turf/turf');
+
+const generateAggregates = (geojson) => {
+  const { features } = geojson;
```

Comment:
Ideally we really want to use the 'getPolygons' method in utilites in the start so we ensure we are consistent.

In general when working with geojson data, it's almost always best to split everything out into polygons when working with it so you don't have a bunch of edge cases from multipart features

Then you can combine it again once you're done